### DISCUSSION TITLE
Dosya Yolları

### DISCUSSION THREAD

**Discussion by guvenrdvan** [gusanmaz/BMB212-Algorithms-2223#23] [Q&A] [answered]
Bizim kendi kök dizinde yollar mesela csv dosyaları kök dizininde bir dosyanın içinde bulunuyor. Kodu derlerken ve çalıştırırken yolu ona göre veriyorum ve sıkıntı çıkmadan çalışıyor. Bu şekilde olduğu halde Repo'da sadece Student.java ve Main.java ve öğrenci numarası bulunucak dimi?

**Comment by gusanmaz** [comment, marked as answer]
Evet.
